### ISSUE TITLE
exame.abril.com.br - anti adblocking script

### ISSUE THREAD

**Issue by KozelElena** [AdguardTeam/AdguardFilters#4897]
[//]: # (***You can leave the strings with "[//]:" They will not be added to the issue text)
[//]: # (***Строки, которые начинаются с "[//]:" можно не удалять. Они не будут видны)

***Description***:
* **Current behaviour**: 

Adguard Content Blocker is noticed by the anti adblocker script in the Samsung Browser.

[//]: # (Tell us what happens)
[//]: # (Опишите проблему)

Screenshot:

[//]: # (use the following code. Replace exciting URL by your)
[//]: # (используйте данный код. Замените существующий адрес адресом вашего скриншота)
<details>

![image](https://cloud.githubusercontent.com/assets/17320558/25555458/9c2dccdc-2cef-11e7-8111-616c19528ef3.png)

</details>

***Steps to reproduce the problem***:

http://exame.abril.com.br/brasil/quem-vai-parar-na-greve-do-dia-28-em-sao-paulo/

[//]: # (Describe, how to reproduce the issue)
[//]: # (Перечислите шаги для воспроизведения проблемы)

***System configuration***

Information | value
--- | ---
Operating system: | Android
Browser: | Samsung
Adguard filters: | English, Spyware, Annoyances, Spanish/Portugese, Mobile ads

[//]: # (***Filters list - comma separated list, or filter names outside the table)
[//]: # (***Список фильтров - названия через запятую, или список вне таблицы)

Ticket ID(if exists): 1548516

[//]: # (This template is for problem reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

**Comment by seanl-adg** [comment]
Not reproduced.

**Comment by Alex-302** [comment]
Also not reproduced. Please check again after filters update.

**Comment by iregados** [comment]
Still happening here after updating the filters.

I made a video that shows the detection when I repported this issue and you check it at the link down here.
If needded I can make another one with the updated filters, but it still looking the same.

Video link:
https://drive.google.com/open?id=0B_UVYX5DCUlONUdneVlISUdpWlBRdWZGbmJVMUZTZ0tQbnlF

**Comment by Alex-302** [comment]
Still can't reproduce in Samsung browser with content blocker(Eng, Spyware, Social, Spanish filters)

**Comment by iregados** [comment]
I tried to update the filters again and it still happening.

Then I've enabled the social media filter, as it's enabled in the list you provided (i got it disabled before) and then the message didnt show.

To be sure I've disabled the social media filter and tried again to open the website and now it's not showing anymore (that's pretty akward to be honest..maybe a cache problem?)

Anyway I think it's solved, thanks for your work!
